src/beamlime/offline/data_feeder.py

```diff
@@ -0,0 +1,52 @@
+import asyncio
+
+import numpy as np
+from colorama import Fore, Style
+from PIL import Image
+from PIL.ImageOps import flip
+
+from ..core.application import BeamlimeApplicationInterface
+from ..resources.images import load_icon_img
+from ..resources.images.generators import fake_2d_detector_img_generator
+
+
+class Fake2dDetectorImageFeeder(BeamlimeApplicationInterface):
+    detector_size = None
+    num_frame = None
+
+    def __init__(
+        self, config: dict, verbose: bool = False, verbose_option: str = Fore.BLUE
+    ) -> None:
+        super().__init__(config, verbose, verbose_option)
+
+    def parse_config(self, config: dict) -> None:
+        if "detector-size" not in config:
+            config["detector-size"] = (64, 64)
```

Comment:
Seems odd that this modifies the external config dict. Is the value written here used outside of this class?

Reply:
Yes, I wanted to save the configuration along with the `log`s.
But indeed it might cause unexpected effect on other applications if each one of `workflow`s becomes an application...
I'll update this part.

Reply:
I opened the issue #34 so it'll be resolved there.